src/components/pages/Robinett/index.jsx

```diff
@@ -0,0 +1,287 @@
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+// Router
+import { Link } from "react-router-dom";
+
+//> Additional
+// React Helmet
+import { Helmet } from "react-helmet";
+
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import {
+  MDBContainer,
+  MDBRow,
+  MDBCol,
+  MDBCard,
+  MDBCardBody,
+  MDBCardFooter,
+  MDBBtn,
+  MDBIcon,
+} from "mdbreact";
+import MDBSortable from "mdb-react-sortable";
+
+//> Components
+import { Egg } from "../../atoms";
+
+//> CSS
+import "./robinett.scss";
+
+//> Data
+const Item = (props) => <span className={props.className}>{props.title}</span>;
```

Comment:
According to ESLINT arrow functions are no capitalised.
```suggestion
const item = (props) => <span className={props.className}>{props.title}</span>;
```
Btw do not forget to change the function call parts in the code.

Reply:
According to ESLINT, EsLiNt is spelled ESLint. Uff

Reply:
This is an component element, so it's meant to be uppercase. (See how it is called in line 36)

Reply:
I see. Nvm.